packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -89,7 +91,8 @@ export const pollAppLogs = async ({
 
         // eslint-disable-next-line no-await-in-loop
         await useConcurrentOutputContext({outputPrefix: log.source, stripAnsi: false}, async () => {
-          if (log.event_type === 'function_run') {
+          // Use only log.log_type after https://github.com/Shopify/partners/pull/55178
+          if ((log.log_type === undefined ? log.event_type : log.log_type) === LOG_TYPE_FUNCTION_RUN) {
```

Comment:
Nit: We can use the `??` operator to simplify this 😄 

```suggestion
          if ((log.log_type ?? log.event_type) === LOG_TYPE_FUNCTION_RUN) {
```